index.js

```diff
@@ -0,0 +1,158 @@
+import * as core from "@actions/core";
+import { LinearClient } from "@linear/sdk";
+
+const ESCAPE = {
+  ">": "&gt;",
+  "<": "&lt;",
+  "&": "&amp;",
+  "\r\n": " ",
+  "\n": " ",
+  "\r": " ",
+};
+const ESPACE_REGEX = new RegExp(Object.keys(ESCAPE).join("|"), "gi");
+
+const getBugs = async (linearClient, stateIds, label) => {
+  const issues = await linearClient.issues({
+    filter: {
+      labels: {
+        and: [{ name: { eq: "Bug" } }, label ? { name: { eq: label } } : {}],
```

Comment:
can we make it include tickets with either "bug" or "release blocker"? or have a separate section for release blockers? we cannot use the "bug" label for both, because it would mess up bug stats for Shringar.

Reply:
Done.